specification/applicationinsights/resource-manager/readme.md

```diff
@@ -222,6 +222,16 @@ directive:
   
 ```
 
+### Tag: package-2021-08-01
```

Comment:
> package-2021-08-01 [](http://example.com/codeflow?start=8&length=19)

should also have a tag which consist the all existing published resourceType, like tag=package-preview-2020-06. maybe call workbooks changes xxx-only?